bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_rdataframe.py

```diff
@@ -96,3 +97,7 @@ def pythonize_rdataframe(klass, name):
         klass.AsNumpy = RDataFrameAsNumpy
 
     return True
+
+# Add MakeNumpyDataFrame feature as free function to the ROOT module
```

Comment:
why can't this be a regular binding to a C++ factory function?

Reply:
Because the input is a `PyObject*` and we need to process it with `ROOT.AsRVec` internally, we need a free function in the module.